extensions/wikia/Insights/scripts/InsightsLoopNotificationTracking.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * LoopNotificationTracking for Insights extension
+ * Module sends tracking record to google analytics and internal analytics
+ * for events related to BannerNotification appearance and clicks.
+ *
+ * checks for insights url param to log specific insight type
+ * checks for item_status url param to log action status (fixed/notfixed)
+ */
+define('ext.wikia.Insights.LoopNotificationTracking',
+	['jquery', 'wikia.tracker', 'wikia.querystring'],
+	function ($, tracker, Querystring) {
+		'use strict';
+
+		var qs = new Querystring(),
+			insightType = qs.getVal('insights', null),
+			itemStatus = qs.getVal('item_status', null);
```

Comment:
I know part with VE is not finished now but we need to consider how to pass this param when VE is enabled